index.html

```diff
@@ -218,19 +218,19 @@ <h1 class="title">About Myslef</h1>
       <h3 class="con">Let's Connect</h3>
       <ul class="social_icons">
         <li>
-          <a href="#"><img src="images/twit.png" alt="" /></a>
+          <a href="#"><img src="images/twit.png" alt="twitter" /></a>
         </li>
         <li>
-          <a href="#"><img src="images/linke.png" alt="" /></a>
+          <a href="#"><img src="images/linke.png" alt="linkedin" /></a>
         </li>
         <li>
-          <a href="#"><img src="images/am.png" alt="" /></a>
+          <a href="#"><img src="images/am.png" alt="Medium" /></a>
         </li>
         <li>
-          <a href="#"><img src="images/github.png" alt="" /></a>
+          <a href="#"><img src="images/github.png" alt="github" /></a>
         </li>
         <li>
-          <a href="#"><img src="images/finger.png" alt="" /></a>
+          <a href="#"><img src="images/finger.png" alt="angellist" /></a>
```

Comment:
Good job in adding alt tags, this will make it easier for users with low bandwidth to know what an image is about.